ansible/roles/csm.cn.ntp/meta/main.yml

```diff
@@ -0,0 +1,4 @@
+# Copyright 2019, Cray Inc. All Rights Reserved.
+# Role dependencies are defined in this file
+---
+dependencies: []
```

Comment:
Are any of these `meta` directories used? Wondering if we can clean them up, removing them so there's less "stuff" to look at.